depends/packages/native_clang.mk

```diff
@@ -1,22 +1,28 @@
 package=native_clang
-$(package)_major_version=13
-$(package)_version=13.0.1
+# To update the Clang compiler:
+# - Change the versions below, and the MSYS2 version in libcxx.mk
+# - Run the script ./contrib/devtools/update-clang-hashes.sh
+# - Manually fix the versions for packages that don't exist (the LLVM project
+#   doesn't uniformly cut binaries across releases).
+# The Clang compiler should use the same LLVM version as the Rust compiler.
+$(package)_major_version=14
+$(package)_version=14.0.6
 $(package)_download_path=https://github.com/llvm/llvm-project/releases/download/llvmorg-$($(package)_version)
-$(package)_download_path_linux=https://github.com/llvm/llvm-project/releases/download/llvmorg-$($(package)_version)
-$(package)_download_file_linux=clang+llvm-$($(package)_version)-x86_64-linux-gnu-ubuntu-18.04.tar.xz
+$(package)_download_path_linux=https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.0
+$(package)_download_file_linux=clang+llvm-14.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz
```

Comment:
Why do these not agree with `$(package)_version`?

Reply:
Because the LLVM project does not always publish packages for every version. That's why the comment at the top of the file says "Run the (new) script, then fix up the Makefile in the ways the script tells you to". The solution is for upstream to automate how they build packages, which I'd attempted to help them with but ran into problems with their packaging system.